test/unit/pdf_find_utils_spec.js

```diff
@@ -0,0 +1,49 @@
+/* Copyright 2018 Mozilla Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { CharacterType, getCharacterType } from '../../web/pdf_find_utils';
+
+describe('pdf_find_utils', function() {
+  describe('getCharacterType', function() {
+    it('gets expected character types', function() {
+      const characters = {
+        'A': CharacterType.ALPHA_LETTER,
+        'a': CharacterType.ALPHA_LETTER,
+        '0': CharacterType.ALPHA_LETTER,
+        '5': CharacterType.ALPHA_LETTER,
+        '\xC4': CharacterType.ALPHA_LETTER, // 'Ä'
+        '\xE4': CharacterType.ALPHA_LETTER, // 'ä'
+        '_': CharacterType.ALPHA_LETTER,
+        ' ': CharacterType.SPACE,
+        '\t': CharacterType.SPACE,
+        '\r': CharacterType.SPACE,
+        '\n': CharacterType.SPACE,
+        '\xA0': CharacterType.SPACE,
+        '-': CharacterType.PUNCT,
+        ',': CharacterType.PUNCT,
+        '.': CharacterType.PUNCT,
+        ';': CharacterType.PUNCT,
+        ':': CharacterType.PUNCT,
```

Comment:
Would it be possible to add a few more tests here for the missing character types, namely:

- HAN_LETTER
- KATAKANA_LETTER
- HIRAGANA_LETTER
- HALFWIDTH_KATAKANA_LETTER
- THAI_LETTER

I think we can simply find them with Wikipedia or perhaps the Mozilla test suite. It would be good to have test coverage for those too.

Reply:
> Would it be possible to add a few more tests here for the missing character types [...]

Of course; this was just me being lazy and not wanting to spend to much time on it until there was feedback on the overall approach :-)

---

> If you would like to add it, the Dutch translation for the new string is:
> find_entire_word_label=Hele woorden

Thanks, it's been added to the patch.